src/main/scala/beam/utils/InputConsistencyCheck.scala

```diff
@@ -0,0 +1,47 @@
+package beam.utils
+
+import beam.agentsim.agents.vehicles.BeamVehicleType
+import beam.sim.config.BeamConfig
+import org.matsim.api.core.v01.Id
+
+import scala.collection.mutable.ListBuffer
+
+object InputConsistencyCheck {
+
+  /**
+    *  Right now we need to check if the values from
+    *
+    *    beam.agentsim.agents.rideHail.initialization.procedural.vehicleTypeId
+    *    beam.agentsim.agents.vehicles.dummySharedCar.vehicleTypeId
+    *
+    *  are present in vehicle types from file
+    *
+    *    beam.agentsim.agents.vehicles.vehicleTypesFilePath
+    */
+  def checkVehicleTypes(
+    vehicleTypes: Set[Id[BeamVehicleType]],
+    rideHailTypeId: String,
+    dummySharedCarTypeId: String
+  ): List[String] = {
+    val errors = ListBuffer[String]()
+    val stringTypes = vehicleTypes.map(_.toString)
+    if (!stringTypes.contains(rideHailTypeId)) {
+      errors.append("beam.agentsim.agents.rideHail.initialization.procedural.vehicleTypeId is not in vehicleTypes")
```

Comment:
To better help debug the error this should list the actual id found. And also should probably list the types tried maybe?

Reply:
done